src/controllers/auth/login.js

```diff
@@ -0,0 +1,11 @@
+export const login = (res) => {
+  try {
+    return res.status(200).send({
+      message: 'Logging in...',
+    });
+  } catch (error) {
+    return res.status(400).send({
+      error: error.message,
+    });
```

Comment:
@davydocsurg now that you've added logic, you'll need write a test for the 200 error response